python/src/opendp/mod.py

```diff
@@ -651,6 +666,7 @@ def _depends_on(self, *args):
 class PartialConstructor(object):
     def __init__(self, constructor):
         self.constructor = constructor
+        self.__opendp_dict__ = {}
```

Comment:
Might work if constructor was in the dict, but constructor not serializable.

Reply:
Does this need to exist at all?